services/explorer-api/src/modules/address/address.service.ts

```diff
@@ -101,11 +102,13 @@ export class AddressService {
   }
 
   public async getBlocksProduced(address: string): Promise<BlocksCount> {
+    const parsedAddress = toChecksumAddress(address);
+
     const [{ pbfts_mined }] = await this.pbftRepository.query(
-      `SELECT COUNT(distinct hash) as pbfts_mined from pbfts WHERE LOWER(miner) = lower('${address}')`
+      `SELECT COUNT(distinct hash) as pbfts_mined from pbfts WHERE LOWER(miner) = lower('${parsedAddress}')`

```

Comment:
If we search `lower` here the conversion to checksum is redundant

Reply:
Yeah but checksum is doing something different, not changing them to lowercase. I would keep both just to be safe.